src/sentry/tasks/weekly_reports.py

```diff
@@ -155,14 +167,20 @@ def prepare_organization_report(
     set_tag("org.slug", organization.slug)
     set_tag("org.id", organization_id)
     ctx = OrganizationReportContext(timestamp, duration, organization)
+    has_escalating_issues = features.has("organizations:escalating-issues", organization)
 
     # Run organization passes
     with sentry_sdk.start_span(op="weekly_reports.user_project_ownership"):
         user_project_ownership(ctx)
     with sentry_sdk.start_span(op="weekly_reports.project_event_counts_for_organization"):
         project_event_counts_for_organization(ctx)
-    with sentry_sdk.start_span(op="weekly_reports.organization_project_issue_summaries"):
-        organization_project_issue_summaries(ctx)
+
+    if has_escalating_issues:
```

Comment:
I think this should be part of the issue states project?

Reply:
is there a different flag for that?

Reply:
`organizations:issue-states`